docs/applications.md

```diff
@@ -1,29 +1,29 @@
-# Child applications
+# registered applications
```

Comment:
Nit: capitalization. There's a couple more in here as well that need to be capitalized.